### DISCUSSION TITLE
Delete uploaded picture

### DISCUSSION THREAD

**Discussion by ghost** [streetcomplete/StreetComplete#4280] [Q&A] [answered]
Hello,
I created a note with StreetComplete where I also added a picture of the entrance of a building. The thing is that I'm visible on the image which I didn't see when I was outside. Now I would like someone to delete this image from the server but I don't know whom I can contact and I don't want to link the image here.

Can somebody here help me with this problem?

Greetings,
Christoph

**Comment by HolgerJeromin** [comment, marked as answer]
cc @westnordost emailadress listed here https://wiki.openstreetmap.org/wiki/User:Westnordost

If the note is closed the image will be deleted a few days (14?) later.

**Reply by ghost** [reply]
Thank you very much! I wasn't able to find the email address. And it's good to know that the image will automatically be deleted after closing the note.

**Reply by westnordost** [reply]
Just checked, it's set to be seven days (it's not deleted immediately after in case the note is reopened for one reason or another).
